setup.py

```diff
@@ -27,5 +27,11 @@
         'Programming Language :: Python :: 3.4',
     ],
     packages=find_packages(),
-    install_requires=['fauxfactory', 'pyxdg', 'requests>=2.7', 'setuptools'],
+    install_requires=[
+        'fauxfactory',
+        'inflector',
```

Comment:
This is a new dependency. The "inflector" library contains a port of the pluralization engine that Ruby on Rails uses. The use of this pluralization engine makes NailGun *much* more capable when dealing with server responses.